src/contracts/pods/EigenPod.sol

```diff
@@ -107,6 +111,9 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
     /// @notice Emitted when podOwner enables restaking
     event restakingActivated(address indexed podOwner);
+
+    /// @notice Emitted when ETH is received via the receive fallback
+    event nonBeaconChainETHReceived(uint256 amountReceived);
```

Comment:
might want an event for withdrawing? idk though

Reply:
Yep, this is a good idea

Reply:
done in this commit https://github.com/Layr-Labs/eigenlayer-contracts/pull/127/commits/fcfed5c7395a42faabb89e04a0ff374d0b0bbab6